doc/classes/EditorSettings.xml

```diff
@@ -577,6 +577,17 @@
 		<member name="interface/inspector/show_low_level_opentype_features" type="bool" setter="" getter="">
 			If [code]true[/code], display OpenType features marked as [code]hidden[/code] by the font file in the [Font] editor.
 		</member>
+		<member name="interface/multi_window/enable" type="bool" setter="" getter="">
+			If [code]true[/code], the multi window support in editor is enabled. The following panels can become dedicated windows (made floating): Docks, Script editor, and Shader editor.
+			When [code]false[/code], modal windows and tooltips are separate windows and displayable outside the editor window.
```

Comment:
This line is misleading. Windows and tooltips are separate regardless of this setting.